### PR TITLE
Fix Dynamic BG

### PR DESCRIPTION
**Pull request by officiallygod**

### CODE REVIEW

File: src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js [hunk 1]
```diff
@@ -9,23 +9,26 @@ import classnames from 'classnames';
...
  */
-import { forwardRef, useEffect } from '@wordpress/element';
+import { forwardRef, useEffect, useContext } from '@wordpress/element';
 // eslint-disable-next-line  @wordpress/no-unsafe-wp-apis
 import { __unstableUseNavigateRegions as useNavigateRegions } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { useMergeRefs } from '@wordpress/compose';
-// import bgAiImg from '../../static/images/ai_bg.png';
+// Needs to explicitly imported to be added dynamically
+import AIImg from '../../static/images/sitegen/ai_bg.png';
```
Comment (arunshenoy99) [inline src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js:18]: Should we use camel case for `AIImg`? Should we rename it to `aiBg`?

File: src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js [hunk 2]
```diff
@@ -9,23 +9,26 @@ import classnames from 'classnames';
...
-		// 	document.querySelector(
-		// 		'.nfd-onboarding-skeleton--sitegen'
-		// 	).style.backgroundImage = `url('${ bgAiImg }')`;
-		// };
+		const mainImage = new Image();
+		mainImage.src = AIImg;
+		mainImage.onload = () => {
+			document.querySelector(
+				'.nfd-onboarding-skeleton--sitegen'
+			).style.background = isDarkMode ? `url('${ AIImg }')` : '#ededed';
```
Comment (arunshenoy99) [inline src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js:30]: Should we ensure that the element exists before accessing the `style` property? Should we remove the hardcoded `#ededed` and replace with a variable?

**Review by arunshenoy99 — APPROVED** [review]
LGTM overall. Added minor non-blocking comments.

File: src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js (hunk 2 above)
Comment (officiallygod) [reply src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js]: Fixed.

File: src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js (hunk 1 above)
Comment (officiallygod) [reply src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js]: Fixed.

**Review by arunshenoy99 — APPROVED** [review]

**Pull request merged**